### DISCUSSION TITLE
Can this be modularized (java 9 modules)?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [sqids/sqids-java#16] [General]
This would require bump to Java 9 at least (better yet - java 11). I can submit pull request for this?

If you insist staying on Java 8 and do not want proper module definition, at least put this entry in the jar manifest:
`Automatic-Module-Name: org.sqids`

**Comment by 0x3333** [comment]
In 2023 1/3 of all Java projects in production were using Java 8. Submit a PR adding the jar manifest.
